WalletWasabi.Fluent/Views/Wallets/WalletView.axaml

```diff
@@ -69,84 +73,158 @@
+          </i:Interaction.Behaviors>
+          <ItemsControl Items="{Binding Tiles}"
+                        Classes.smallLayout="{Binding IsSmallLayout}"
+                        Classes.normalLayout="{Binding IsNormalLayout}"
+                        Classes.wideLayout="{Binding IsWideLayout}">
+            <ItemsControl.Styles>
+              <Style Selector="ItemsControl.smallLayout">
+                <Setter Property="HorizontalAlignment" Value="Left"/>
+                <Setter Property="VerticalAlignment" Value="Top"/>
+                <Setter Property="Margin" Value="0 0"/>
+                <Setter Property="Padding" Value="0,0,0,10"/>
+              </Style>
+              <Style Selector="ItemsControl.normalLayout">
+                <Setter Property="HorizontalAlignment" Value="Left"/>
+                <Setter Property="VerticalAlignment" Value="Top"/>
+                <Setter Property="Margin" Value="0 0"/>
+                <Setter Property="Padding" Value="0,0,0,10"/>
+              </Style>
+              <Style Selector="ItemsControl.wideLayout">
+                <Setter Property="HorizontalAlignment" Value="Left"/>
+                <Setter Property="VerticalAlignment" Value="Top"/>
+                <Setter Property="Margin" Value="0 0"/>
+                <Setter Property="Padding" Value="0,0,10,0"/>
+              </Style>
+              <Style Selector="ItemsControl > ContentPresenter" x:DataType="tiles:TileViewModel">
+                <Setter Property="Grid.Column" Value="{Binding Column}" />
+                <Setter Property="Grid.Row" Value="{Binding Row}" />
+                <Setter Property="Grid.ColumnSpan" Value="{Binding ColumnSpan}" />
+                <Setter Property="Grid.RowSpan" Value="{Binding RowSpan}" />
+                <Setter Property="IsVisible" Value="{Binding IsVisible}" />
+              </Style>
+            </ItemsControl.Styles>
+            <ItemsControl.ItemsPanel>
+              <ItemsPanelTemplate>
+                <c:BindableGrid ColumnDefinitionsSource="{Binding CurrentLayout.ColumnDefinitions}"
+                                RowDefinitionsSource="{Binding CurrentLayout.RowDefinitions}"
+                                ShowGridLines="False" />
+              </ItemsPanelTemplate>
+            </ItemsControl.ItemsPanel>
+            <ItemsControl.ItemTemplate>
+              <locator:ViewLocator />
+            </ItemsControl.ItemTemplate>
+          </ItemsControl>
+        </ScrollViewer>
+        <Border ClipToBounds="False"
+                CornerRadius="5"
+                Background="{DynamicResource TileRegionColor}"
+                Margin="6"
+                Padding="16"
+                BoxShadow="{DynamicResource InfoBoxShadow}"
+                IsVisible="{Binding IsEmptyWallet}">
+          <StackPanel>
+            <Label Content="Since this is a empty wallet, there isn't a lot going on here. Click an option below to receive Bitcoin."
```

Comment:
tACK a27096e, consider

```suggestion
            <Label Content="Since this is a empty wallet, there isn't a lot going on here. Click below to receive Bitcoin."
```

Reply:
Updated label text